src/var.jl

```diff
@@ -43,12 +43,13 @@ end
 
 struct PolyVar{C} <: AbstractVariable
     id::Int
-    name::AbstractString
+    name::String
```

Comment:
@blegat Can you tell me why this was previously `AbstractString`? I don't see why we should put an abstract type there...

Reply:
Don't remember, `String` is better I agree :)